Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -197,89 +198,123 @@ void GeneralWidget::AddDescriptions()
 // We need QObject::tr
 #if defined(_WIN32)
   static const char TR_BACKEND_DESCRIPTION[] = QT_TR_NOOP(
-      "Selects which graphics API to use internally.\n\nThe software renderer is extremely "
+      "Selects which graphics API to use internally.<br /><br />The software renderer is extremely "
```

Comment:
Does `<br>` not work? There's no reason to use the XHTML compatible form here.

If `<br>` does work, please change all the other occurrences of `<br />` for readability reasons.